mobile/library/cc/engine_builder.h

```diff
@@ -60,6 +60,10 @@ class EngineBuilder {
   EngineBuilder& enableDrainPostDnsRefresh(bool drain_post_dns_refresh_on);
   EngineBuilder& enforceTrustChainVerification(bool trust_chain_verification_on);
   EngineBuilder& enablePlatformCertificatesValidation(bool platform_certificates_validation_on);
+  // Adds an RTDS layer to default config
```

Comment:
Can we also add that setAggregatedDiscoveryService must be called before addRtdsLayer is called?